docs/drilldowns/explore-underlying-data.asciidoc

```diff
@@ -0,0 +1,45 @@
+[[explore-underlying-data]]
+== Explore data underlying visualizations
```

Comment:
Maybe re-phrase to "Explore the underlying data powering your visualizations"?

The way it's phrased right now makes it confusing to figure out which noun the word "underlying" belongs to.

Reply:
#### Title

How about this as a title:  Explore the underlying data for a visualization

#### Title abbreviation

Include this below the title, so a shorter version appears in the TOC:

++++
<titleabbrev>Explore the underlying data</titleabbrev>
++++

#### How to make the doc visible

In dashboard.asciidoc, after this line:

include::{kib-repo-dir}/drilldowns/drilldowns.asciidoc[]

add this line:

include::{kib-repo-dir}/drilldowns/explore-underlying-data.asciidoc[]

Reply:
Done.